app/components/support_interface/ucas_match_action_component.html.erb

```diff
@@ -7,7 +7,7 @@
       <p class="govuk-body">
         <%= action_details %>
       </p>
-      <% if @match.action_needed? %>
+      <% if @match.action_needed? && @match.next_action == :ucas_withdrawal_requested %>
```

Comment:
Maybe define this as a method on the match — `match.requires_manual_action?` or sth?